### PR TITLE
Add a temporary workaround for an issue #224

### PR DESCRIPTION
**Pull request by voronkovich**
See https://github.com/symfony/symfony-demo/issues/224

### CODE REVIEW

File: src/CodeExplorerBundle/Twig/SourceCodeExtension.php [hunk 1]
```diff
@@ -99,15 +99,23 @@ private function getCallableReflector($callable)
...
     {
+        // Twig templates are not always stored in files, and so there is no
+        // API to get the filename from a template name in a generic way.
+        // The logic used here works only for templates stored in app/Resources/views
+        // and referenced via the "filename.html.twig" notation, not via the "::filename.html.twig"
+        // one or stored in bundles. This is enough for the needs of the demo app.
+        $filePath   = $this->kernelRootDir.'/Resources/views/'.$template->getTemplateName();
+        $sourceCode = $template->getSource();
+
+        // Temporary workaround for https://github.com/symfony/symfony-demo/issues/224
```
Comment (javiereguiluz) [inline src/CodeExplorerBundle/Twig/SourceCodeExtension.php]: Maybe we can change this URL and link to the Twig issue report: `https://github.com/twigphp/Twig/issues/2011`

File: src/CodeExplorerBundle/Twig/SourceCodeExtension.php [hunk 2]
```diff
@@ -99,15 +99,23 @@ private function getCallableReflector($callable)
 
     private function getTemplateSource(\Twig_Template $template)
     {
+        // Twig templates are not always stored in files, and so there is no
+        // API to get the filename from a template name in a generic way.
+        // The logic used here works only for templates stored in app/Resources/views
+        // and referenced via the "filename.html.twig" notation, not via the "::filename.html.twig"
+        // one or stored in bundles. This is enough for the needs of the demo app.
+        $filePath   = $this->kernelRootDir.'/Resources/views/'.$template->getTemplateName();
```
Comment (stof) [inline src/CodeExplorerBundle/Twig/SourceCodeExtension.php]: ``=`` signs should not be aligned

File: src/CodeExplorerBundle/Twig/SourceCodeExtension.php (hunk 1 above)
Comment (voronkovich) [inline src/CodeExplorerBundle/Twig/SourceCodeExtension.php]: @javiereguiluz I've changed the issue url

File: src/CodeExplorerBundle/Twig/SourceCodeExtension.php (hunk 2 above)
Comment (voronkovich) [inline src/CodeExplorerBundle/Twig/SourceCodeExtension.php]: Fixed!

**Comment by javiereguiluz** [comment]
I've tested this locally and it totally solves the issue for me 👍 

@voronkovich thank you very much for working on this nasty bug!

**Comment by yceruto** [comment]
:+1:

**Pull request closed without merging**